doc/rados/operations/placement-groups.rst

```diff
@@ -700,134 +705,146 @@ For example:
+PGs will be processed in their default order. As in the case of adding the
+``force`` flag, this affects only those PGs that are still queued but does not
+affect currently processed PGs.
 
-The "force" flag is cleared automatically after recovery or backfill of group
-is done.
+The ``force`` flag is cleared automatically after the recovery or backfill of
+the PGs is complete.
 
-Similarly, you may use the following commands to force Ceph to perform recovery
-or backfill on all placement groups from a specified pool first:
+Similarly, to instruct Ceph to prioritize all PGs from a specified pool (that
+is, to perform recovery or backfill on those PGs first), run one or both of the
+following commands:
 
 .. prompt:: bash #
 
    ceph osd pool force-recovery {pool-name}
    ceph osd pool force-backfill {pool-name}
 
-or:
+These commands can also be cancelled. To revert to the default order, run one
+or both of the following commands:
 
 .. prompt:: bash #
 
    ceph osd pool cancel-force-recovery {pool-name}
    ceph osd pool cancel-force-backfill {pool-name}
 
-to restore to the default recovery or backfill priority if you change your mind.
-
-Note that these commands could possibly break the ordering of Ceph's internal
-priority computations, so use them with caution!
-Especially, if you have multiple pools that are currently sharing the same
-underlying OSDs, and some particular pools hold data more important than others,
-we recommend you use the following command to re-arrange all pools's
-recovery/backfill priority in a better order:
+.. note:: These commands can break the order of Ceph's internal priority
+   computations, so use them with caution! If you have multiple pools that are
+   currently sharing the same underlying OSDs, and if the data held by certain
+   pools is more important than the data held by other pools, then we recommend
+   that you run a command of the following form to arrange a custom
+   recovery/backfill priority for all pools:
 
 .. prompt:: bash #
 
    ceph osd pool set {pool-name} recovery_priority {value}
 
-For example, if you have 10 pools you could make the most important one priority 10,
-next 9, etc. Or you could leave most pools alone and have say 3 important pools
-all priority 1 or priorities 3, 2, 1 respectively.
+For example, if you have 10 pools you could make the most important one
+priority 10, next 9, etc. Or you could leave most pools alone and have say 3
+important pools all priority 1 or priorities 3, 2, 1 respectively.
 
-Revert Lost
-===========
+Reverting Lost RADOS Objects
+============================
 
-If the cluster has lost one or more objects, and you have decided to
+If the cluster has lost one or more RADOS objects and you have decided to
 abandon the search for the lost data, you must mark the unfound objects
-as ``lost``.
+``lost``.
+
+If every possible location has been queried and certain RADOS objects are still
```

Comment:
has been queried and all OSDs are `up` and `in`,

Reply:
Accepted.